### PR TITLE
[SCB-716] Implementation of Local Service Registry Demo

### PR DESCRIPTION
**Pull request by maheshrajus**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by liubao68** [comment]
retrigger CI

File: demo/demo-local-service-registry/local-registry-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```
Comment (liubao68) [inline demo/demo-local-service-registry/local-registry-client/pom.xml]: Can you move these codes to samples folder？ Java-chassis demo folder is for integration tests . We are planning to change it but got no time to deal this. 
And add a README.md for this sample project to learn what it is do and how to run this sample.

Comment (maheshrajus) [reply demo/demo-local-service-registry/local-registry-client/pom.xml]: ok..i will move and raise PR.

File: samples/local-service-registry/local-registry-client/src/main/resources/microservices/localserv/localservregistry.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+## ---------------------------------------------------------------------------
```
Comment (wujimin) [inline samples/local-service-registry/local-registry-client/src/main/resources/microservices/localserv/localservregistry.yaml]: seems not a good schema id.

File: samples/local-service-registry/local-registry-server/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryserver/LocalRegistryServer.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+package org.apache.servicecomb.samples.localregistry.localregistryserver;
+
+import org.apache.servicecomb.foundation.common.utils.BeanUtils;
+import org.apache.servicecomb.foundation.common.utils.Log4jUtils;
+
+public class LocalRegistryServer {
+  public static void main(String[] args) throws Exception {
+    Log4jUtils.init();
```
Comment (wujimin) [inline samples/local-service-registry/local-registry-server/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryserver/LocalRegistryServer.java]: this sample means:
1.server connect to sc
2.client not connec to sc?

File: samples/local-service-registry/local-registry-client/src/main/resources/microservices/localserv/localservregistry.yaml (hunk 2 above)
Comment (maheshrajus) [reply samples/local-service-registry/local-registry-client/src/main/resources/microservices/localserv/localservregistry.yaml]: ok, i will change schema id

File: samples/local-service-registry/local-registry-server/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryserver/LocalRegistryServer.java (hunk 3 above)
Comment (maheshrajus) [reply samples/local-service-registry/local-registry-server/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryserver/LocalRegistryServer.java]: 1) server will not connect SC
2) client will not connect SC
client will connect server's APIs with help of local registry function.

File: samples/local-service-registry/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  ```
+
+* Add the following code at the beginning and end of the service consumer Main function
+
+```java
+    public class xxxClient {
+      public static void main(String[] args) {
+　　    System.setProperty("local.registry.file", "/yourpath/registry.yaml");
+        //your code
+　　    System.clearProperty("local.registry.file");
```
[the remark this answers is not in the document]
Comment (maheshrajus) [reply samples/local-service-registry/README.md]: This is System property, better to clear after our work.

**Pull request merged**